vignettes/download_github_comments.Rmd

```diff
@@ -377,7 +369,22 @@ tail(all_pr,1)  %>%
   gt(auto_align = FALSE) 
 ```
 
+## Pull Request Review Comments Endpoint
+
```

Comment:
I would consider adding a little more description. Maybe something about how PR review comments are directly tied to lines of code, which can help to identify patterns in collaboration, concerns, etc.